pkg/codegen/utils.go

```diff
@@ -57,25 +56,33 @@ func LowercaseFirstCharacter(str string) string {
 // So, "word.word-word+word:word;word_word~word word(word)word{word}[word]"
 // would be converted to WordWordWordWordWordWordWordWordWordWordWordWordWord
 func ToCamelCase(str string) string {
-	separators := []string{".", "-", "+", ":", ";", "_", "~", " ", "(", ")", "{", "}", "[", "]"}
-	in := []string{str}
-	out := make([]string, 0)
-
-	for _, sep := range separators {
-		for _, inStr := range in {
-			parts := strings.Split(inStr, sep)
-			out = append(out, parts...)
+	separators := "-#@!$&=.+:;_~ (){}[]"
+	s := strings.Trim(str, " ")
+
+	n := ""
+	capNext := true
+	for _, v := range s {
+		if v >= 'A' && v <= 'Z' {
```

Comment:
This change using ASCII character ranges makes one big difference - it no longer works properly with unicode characters. If you do it this way, it might be better to use the unicode package and test for IsUpper, isLower, IsNumber, etc. The previous code carefully avoided that by breaking up what we had into words, and capitalizing each word

Reply:
Done! Thanks for taking a look.